tla/consensus/ccfraft.tla

```diff
@@ -1177,6 +1180,8 @@ DropIgnoredMessage(i,j,m) ==
        \/ /\ leadershipState[i] /= None
         \* .. and it comes from a server outside of the configuration
           /\ \lnot IsInServerSet(j, i)
+          \* We specifically don't ignore RequestVoteRequest messages from unknown nodes
+          /\ m.type \notin {RequestVoteRequest}
```

Comment:
I can't quite remember what we decided wrt cross-linking with comments, but for what it's worth this is [here in the implementation](https://github.com/microsoft/CCF/blob/dbbd981954836f88f9402111d096afc3e27658bf/src/consensus/aft/raft.h#L1688).